### ISSUE TITLE
VictoryBar and tick alignment off

### ISSUE THREAD

**Issue by cleverettgirl** [FormidableLabs/victory#1036]
Hello! First of all, great library! I was really excited when we decided to use it.. I am having a little bit of an issue though, with VictoryBar and tick alignment. I can't figure out what's causing it be funky and i've tried different combinations of domainPadding and tickLabelComponent props, but to no avail.

Picture of when it's not too bad
<img width="943" alt="screen shot 2018-06-29 at 1 52 24 pm" src="https://user-images.githubusercontent.com/16964119/42115276-35989e7c-7bc0-11e8-8633-4e0a111193cf.png">

Picture of when it is..
<img width="821" alt="screen shot 2018-06-29 at 5 17 08 pm" src="https://user-images.githubusercontent.com/16964119/42115325-6202205a-7bc0-11e8-9319-33847d88cdcd.png">

TLDR:
I have a dropdown that changes the date range, sometimes the axisLabel is centered and other times it's not. Here's what I have so far:
https://codesandbox.io/embed/zxjo47jz13
